### ISSUE TITLE
Error when adding AC3259/60

### ISSUE THREAD

**Issue by TheCableGuy99** [kongo09/philips-airpurifier-coap#80]
Hi,

When I try to add this it just says an unknown error occured. The AC3259 is on the supported list but I'm not sure what the /60 is for my model?

I've set a static IP, I can access it on the official mobile app, I just can't add it in HA.

Any idea's please?

Thanks.

**Comment by kongo09** [comment]
Can you please enable logging and post the debug logs?

**Comment by TheCableGuy99** [comment]
Hey,

I'm not 100% sure these are the logs you are looking for but it's the only ones I could find (very new to Home Assistant and setting it up for the first time). I got these from: Settings > System > Logs

`This error originated from a custom integration.

Logger: custom_components.philips_airpurifier_coap.config_flow
Source: custom_components/philips_airpurifier_coap/config_flow.py:211
Integration: Philips AirPurifier (with CoAP) (documentation, issues)
First occurred: 09:56:25 (1 occurrences)
Last logged: 09:56:25

Timeout, host 192.168.10.230 doesn't answer, aborting`

If there are any better logs please let me know where to get them?

Thanks.

**Comment by kongo09** [comment]
This feels like your device is not supporting encrypted CoAP. Do you have access to the command line? If so, could you please try to get a status from your device as described in the README?

**Comment by vszurma** [comment]
I got the same error and after some debugging with wireshark and research I found, that the device is sometimes responding really really slowly. After review of the code i noticed that the COAP Client was instantiated with timeout=25, which is to low because I observed up to one minute delays. I changed the timeout=180 and it worked perfectly with my devices. Maybe this should be the default and maybe much greater.

The models I have are: AC2939 and AC3033

Hope this helps
